### PR TITLE
Add better readme

### PR DESCRIPTION
**Pull request by Sidd-underscore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# `discord.js` Bot Template
+# Elysium
 
-This is a template for a Discord bot. It is written in JavaScript and uses the `discord.js` library. It is designed to be easy to use and easy to modify.
+This is the source code for you favorite AI Discord bot. It is written in JavaScript and uses the `discord.js` library. It is designed to be easy to use and easy to modify.
+
+Includes conmands that allow you to generate and explain images, chat with the assistant and set custom personalities, transcribe text into audio, and so much more!
+
+> AI is powered by [PurGPT](https://purgpt.xyz) and Tolgchu's own money (when PurGPT is offline)
```
Comment (Pukimaa) [inline README.md:7]:
```suggestion
> AI is powered by [PurGPT](https://purgpt.xyz) and my own money (when PurGPT is offline)
```
This should be in first-person.

Comment (Sidd-underscore) [reply README.md:7]: Real, makes sense

Comment (Pukimaa) [reply README.md:7]: > Real, makes sense

Then accept it :3

Comment (Sidd-underscore) [reply README.md:7]: Oh oopsie I forgor

**Comment by Tolga1452** [comment]
sorry i'll close this pr until completely finish the bot

**Pull request closed without merging**